.github/workflows/release.yaml

```diff
@@ -1,63 +1,78 @@
 name: Release
 
 on:
-  push:
-    tags:
-      # we don't want to be triggered for RC releases (e.g. v0.6.0-rc9)
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to release, without prefix v (e.g. 1.1.0)'
+        required: true
+      branch:
+        description: 'The branch to release from (e.g. 1.0.x)'
+        required: true
+        default: 'master'
```

Comment:
I think it will be very useful to allow choose which steps to skip when dispatching the workflow. 
Same for `Build` workflow now that I am thinking about it.

Reply:
answered in the general comment